components/add-member.tsx

```diff
@@ -81,8 +81,7 @@ export default function AddMemberForm({ user }: { user: any }) {
   }
 
   async function onSubmit(data: MemberFormValues) {
-    console.log("DATA", selectedGroups);
-    try {
+    
```

Comment:
```suggestion
    try {
```

add the `try {` back